### PR TITLE
opening the game panel no longer slows the game down

### PR DESCRIPTION
**Pull request by Evankhell561**
opening the game panel no longer automatically displays all available objects.
displaying all objects in the game consumes a lot of resources and slows down the game for a few seconds.
:cl:
tweak: opening the game panel no longer automatically displays all available objects.
/:cl:

### CODE REVIEW

File: html/create_object.html [hunk 1]
```diff
@@ -46,20 +46,19 @@
...
+
 	<script language="JavaScript">
 		var old_search = "";
 		var object_list = document.spawner.object_list;
 		var object_paths = null /* object types */;
 		var objects = object_paths == null ? new Array() : object_paths.split(";");
-		
+
 		document.spawner.filter.focus();
-		populateList(objects);
```
Comment (Evankhell561) [inline html/create_object.html:57]: this is the only change made.

Comment (NanakoAC) [reply html/create_object.html:57]: this requires explanation. What exactly does this do and why remove it?

Comment (Evankhell561) [reply html/create_object.html:57]: when opening the game panel, this automatically causes the list of all objects in the game to be displayed.

this happens for a second during which the game freezes and at the end, you don't search within that list but filter the path you want to find via the search engine.

Comment (Evankhell561) [reply html/create_object.html:57]: In general, this makes the processor cry every time the game panel is opened.

**Pull request merged**